XIVSlothCombo/Combos/RDM.cs

```diff
@@ -643,56 +656,258 @@ const int
+                        {
+                            if (IsOnCooldown(RDM.Manafication)) step++;
+                            else return RDM.Manafication;
+                        }
+
+                        if (step == 7)
+                        {
+                            if (lastComboMove == RDM.Riposte) step++;
+                            else return RDM.EnchantedRiposte;
+                        }
+
+                        if (step == 8)
+                        {
+                            if (IsOnCooldown(RDM.Fleche)) step++;
+                            else return RDM.Fleche;
+                        }
+
+                        if (step == 9)
+                        {
+                            if (lastComboMove == RDM.Zwerchhau) step++;
+                            else return RDM.EnchantedZwerchhau;
+                        }
+
+                        if (step == 10)
+                        {
+                            if (IsOnCooldown(RDM.ContreSixte)) step++;
+                            else return RDM.ContreSixte;
+                        }
+
+                        if (step == 11)
+                        {
+                            if (lastComboMove == RDM.Redoublement || gauge.ManaStacks == 3) step++;
+                            else return RDM.EnchantedRedoublement;
+                        }
+
+                        if (step == 12)
+                        {
+                            if (GetRemainingCharges(RDM.Corpsacorps) < 2) step++;
+                            else return RDM.Corpsacorps;
+                        }
+
+                        if (step == 13)
+                        {
+                            if (GetRemainingCharges(RDM.Engagement) < 2) step++;
+                            else return RDM.Engagement;
+                        }
+
+                        if (step == 14)
+                        {
+                            if (lastComboMove == RDM.Verholy) step++;
+                            else return RDM.Verholy;
+                        }
+
+                        if (step == 15)
+                        {
+                            if (GetRemainingCharges(RDM.Corpsacorps) < 1) step++;
+                            else return RDM.Corpsacorps;
+                        }
+
+                        if (step == 16)
+                        {
+                            if (GetRemainingCharges(RDM.Engagement) < 1) step++;
+                            else return RDM.Engagement;
+                        }
+
+                        if (step == 17)
+                        {
+                            if (lastComboMove == RDM.Scorch) step++;
+                            else return RDM.Scorch;
+                        }
+
+                        if (step == 18)
+                        {
+                            if (lastComboMove == RDM.Resolution) step++;
+                            else return RDM.Resolution;
+                        }
+
```

Comment:
I see what you're doing here, but honestly should this be less of a multistep combo you think? It might be a bit on the automation heavy side, this is an 18 move combo... idk

Reply:
I thought of the steps because we need those skills to be called in a correct order for the opener to work, I agree it goes a bit on the heavy automation side, without the steps we would just call the skills when they are up and that would mess up the opener order or we would need specific control variables to know when to call what in order, and then it would end up at the same place.

If that is too much I can remove it np.